src/externalized/scripting/FunctionsLibrary.h

```diff
@@ -200,6 +232,28 @@ OBJECTTYPE* CreateMoney(const UINT32 amt);
  * @ingroup funclib-items
  */
 void PlaceItem(const INT16 sGridNo, OBJECTTYPE* const pObject, const INT8 bVisibility);
+/**
+ * Adds a recurring events that happens at the same time every day.
+ * @param ubCallbackID strategic event ID
+ * @param uiStartMin the time (minutes of day) that the event
+ * @param uiParam a parameter that will be passed to the event handler
+ */
+void AddEveryDayStrategicEvent_(UINT8 ubCallbackID, UINT32 uiStartMin, UINT32 uiParam);
+
+/**
+ * Adds an one-off strategic event
+ * @param ubCallbackID strategic event ID
+ * @param uiMinStamp earliest time (in world-seconds) that this event will be processed
+ * @param uiParam a parameter that will be passed to the event handler
+ */
+void AddStrategicEvent_(UINT8 ubCallbackID, UINT32 uiMinStamp, UINT32);
+void StartQuest_(UINT8 ubQuestID, std::string sectorID);
+void EndQuest_(UINT8 ubQuest, std::string sectorID);
```

Comment:
These two should also use const references for `sectorID`.

Reply:
This is a function to be invoked from Lua, and the args will go through serialization. Pass-by-reference does not work. We can only pass by pointer or pass by value.

This arg is not to be modified, and it is not supposed to be called frequently. So it is easier passed by value, as it doesn't have potential problems of null pointers.